packages/service/src/glint-service.ts

```diff
@@ -1,7 +1,7 @@
 // This is disabled because vscode-uri is a commonjs module, which causes TS and Eslint to disagree
 // about how it should be imported (╯°□°)╯︵ ┻━┻
 // eslint-disable-next-line import/default
```

Comment:
Is this disabling comment needed anymore? I think that's specifically what 3.0.8 fixes.

Reply:
You are correct.  Will remove.